test/com/google/enterprise/adaptor/fs/MockFileDelegate.java

```diff
@@ -180,19 +177,18 @@ public Path resolveDfsLink(Path doc) throws IOException {
   }
 
   @Override
-  public List<Path> enumerateDfsLinks(Path doc) throws IOException {
-    MockFile file = getFile(doc);
-    if (file.isDfsNamespace()) {
-      ImmutableList.Builder<Path> builder = ImmutableList.builder();
-      for (Path path : file.newDirectoryStream()) {
-        if (isDfsLink(path)) {
-          builder.add(path);
-        }
-      }
-      return builder.build();
-    } else {
+  public DirectoryStream<Path> newDfsLinkStream(Path doc) throws IOException {
+    final MockFile file = getFile(doc);
+    if (!file.isDfsNamespace()) {
```

Comment:
I would flip this code back to its original order, but I don't mind if you left it flipped on purpose. The rearranged code is more consistent with newDirectoryStream in MockFile.